godotproject/Player.tscn

```diff
@@ -413,7 +413,7 @@ position = Vector2( 0.446458, -7.12662 )
 texture = ExtResource( 2 )
 vframes = 6
 hframes = 5
-frame = 3
+frame = 2
```

Comment:
Seems like an accidental change. I'd say to leave it as is. I think it just relates to what the last-played frog animation is.
```suggestion
frame = 3
```